src/main/java/spoon/support/reflect/reference/CtImplicitTypeReferenceImpl.java

```diff
@@ -2,12 +2,18 @@
 
 import spoon.reflect.reference.CtImplicitTypeReference;
 import spoon.reflect.reference.CtReference;
+import spoon.reflect.visitor.CtVisitor;
 
 public class CtImplicitTypeReferenceImpl<R> extends CtTypeReferenceImpl<R>
```

Comment:
Moves to internal package when https://github.com/INRIA/spoon/pull/374 is merged.